banzai/lco.py

```diff
@@ -22,7 +22,7 @@ class LCOObservationFrame(ObservationFrame):
     def get_output_filename(self, runtime_context):
         # TODO add a mode for AWS filenames
         output_directory = os.path.join(runtime_context.processed_path, self.instrument.site,
-                                        self.instrument.name, self.epoch, 'processed')
+                                        self.instrument.camera, self.epoch, 'processed')
```

Comment:
This will break NRES. It may be the right thing to do, but we need to make NRES consistent.

Reply:
@mgdaily Can you comment on this? If this method is to build the AWS daydir, then I think we store the data by instrument code (instrument.name) rather than camera code (instrument.camera), but I am not sure how this is used exactly?

Reply:
Yes, that's correct. This builds the daydir structure for when we write to disk. Name is what we want in this case.